interface/forms/soap/FormSOAP.class.php

```diff
@@ -35,15 +35,20 @@ class FormSOAP extends ORDataObject
      */
```

Comment:
hi @boxlady , may want to create new with `interface/forms/soap_history/FormSOAPHistory.class.php` so preserve legendary SOAP :)